src/style.css

```diff
@@ -0,0 +1,75 @@
+@font-face {
+  font-family: Avenir;
+  src: url(../src/fonts/Avenir-Book.ttf);
+}
+@font-face {
+  font-family: Avenir-Heavy;
+  src: url(../src/fonts/Avenir-Heavy.ttf);
+}
+
+html {
+  font-family: "Avenir" , sans-serif;
+}
+
+.search {
+  display: flex;
+  align-items: center;
+  margin: 20px auto;
+  background: #fff;
+  border: 1px solid #e1e7ed;
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+  width: 100%;
+  border-radius: 4px;
+  box-sizing: border-box;
+}
+
+.big-search {
+  height: 70px;
+}
+
+.small-search {
+  height: 42px;
+}
+
+.big-img-search {
+  padding: 24px 15px 25px 25px;
+}
+
+.small-img-search {
+  width: 11px;
+  height: 11px;
+  vertical-align: middle;
+  padding: 14px 9px 15px 12px;
+}
+
+.big-input-search ,
+.small-input-search {
+  font-family: inherit;
+  border: none;
+}
```

Comment:
Why you decided to disable border? Also you can apply font-family prop to `search` class, no need to duplicate it for this classes

Reply:
in task , when we look on our search , we don't see the input-border , so I decided to disable him . Mb I should to make a white border like `border: 1px solid white;` ?

Reply:
I meant only border
There is a same border for both inputs for all states in figma, check it please. That's why there is no need to disable it